Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:15-buster
+
+EXPOSE 80
+
+RUN mkdir /app
+
+# Copying requered files
+COPY ./nginx.conf.sigil /app
```

Comment:
Is `./<DIRNAME` and `/<DIRNAME` referring to different paths in this case? Is it copying something from the current root directory to the system root directory?

Reply:
Yea its copying stuff from the repo into the container